test_multi_channel_fix.py

```
@@ -0,0 +1,228 @@
+            try:
+                # This calls our fixed _create_bot method
+                bot = gateway._create_bot(
+                    channel_type="test",
+                    token=f"test_token_{i}",
+                    agent=test_agent,
+                    config=config,
+                    ch_cfg=ch_cfg
+                )
+                bots.append(bot)
+                print(f"✓ Successfully created bot {i+1}")
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)